cli/src/action/database/mod.rs

```diff
@@ -23,12 +23,26 @@ use diesel::{connection::Connection as _, pg::PgConnection};
 use splinter::migrations::run_postgres_migrations;
 
 use crate::error::CliError;
+use splinter::admin::store::error::AdminServiceStoreError;
+use splinter::admin::store::yaml::YamlAdminServiceStore;
+use splinter::admin::store::AdminServiceStore;
+use std::env;
```

Comment:
Please rearrange your use statements into blocks in the order std, external crate, internal crate. We try to standardize this format for the use statements.

Reply:
With a new line in between, so the formatter doesn't resort them.

This is one thing we can't (currently) catch with the linter, but is a team preference.

Reply:
Fixed!